src/usage_metrics/scripts/save_kaggle_metrics.py

```diff
@@ -0,0 +1,49 @@
+"""This script pull Kaggle traffic metrics and saves them to a GC Bucket."""
+
+import json
+import logging
+import sys
+from datetime import date
+
+from google.cloud import storage
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+KAGGLE_OWNER = "catalystcooperative"
+KAGGLE_DATASET = "pudl-project"
```

Comment:
I think these (and `BUCKET_NAME`) probably don't need to be module-level variables either, sort of like the github metrics situation.